### DISCUSSION TITLE
Can Control re-use the same models (and preprocessors) from an installed ControlNet extension?

### DISCUSSION THREAD

**Discussion by lbeltrame** [vladmandic/sdnext#2733] [Q&A] [answered]
As the subject says, assuming I point it to (or symlink) the folder that contains them. The reason is that on cloud platforms like Paperspace the space is severely limited (and these models are heavy, even more for SDXL), so I usually download them to `/tmp` and then symlink them to where they should be.  I would want to make sure that Control doesn't download anything that I already have.

**Comment by vladmandic** [comment, marked as answer]
in theory, yes.
control module is different than controlnet extension since it splits different types into subfolders, but each can still be linked.
e.g. models/control/controlnet

**Comment by lbeltrame** [comment]
Thanks, I'll play with it. In the worst case I'll symlink the folder to /tmp and have it download every time needed.

**Reply by vladmandic** [reply]
Symlink will work, you just need to be aware to symlink each of subfolders, not main folder.
